jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandlerCollection.java

```diff
@@ -378,12 +360,15 @@ public String toString()
     private static class Mapping extends Handlers
     {
         private final Map<ContextHandler, Handler> _contextBranches = new HashMap<>();
-        private final Trie<Map.Entry<String, Branch[]>> _pathBranches;
+        private final Index.Mutable<Map.Entry<String, Branch[]>> _pathBranches;
 
-        private Mapping(Handler[] handlers, int capacity)
+        private Mapping(Handler[] handlers)
         {
             super(handlers);
-            _pathBranches = new ArrayTernaryTrie<>(false, capacity);
+            _pathBranches = new Index.Builder<Map.Entry<String, Branch[]>>()
+                .caseSensitive(true)
+                .mutable()
+                .build();
```

Comment:
Need to pass in all the context paths and not have this mutable

Reply:
Done, `ContextHandlerCollection.Mapping` is now immutable.